core/optionsx.h

```diff
@@ -539,6 +539,11 @@
     OPTION_DEFAULT_INTERNAL(uint, opt_mangle, 1,
                             "optimization level on optimizing mangle sequences")
 #endif
+#ifdef AARCH64
+    OPTION_DEFAULT_INTERNAL(bool, build_ldstex, false,
```

Comment:
If this is meant to help users and not just be internal to a few devs, it should be documented in the section on the ldrex..strex problem, right?